omod/src/main/java/org/openmrs/module/dhisconnector/web/controller/DHISConnectorController.java

```diff
@@ -447,6 +442,24 @@ public void postAutomationPage(ModelMap model, HttpServletRequest request) {
 			request.getSession().setAttribute(WebConstants.OPENMRS_MSG_ATTR, response);
 	}
 
+	@RequestMapping(value = "/module/dhisconnector/automation", params = "saveAutomationToggle", method = RequestMethod.POST)
```

Comment:
thanks @Piumal1999  do you need this (saveAutomationToggle) here?

Reply:
Yeah @HerbertYiga . Thanks for the review. 
We have 2 mappings for the path `/module/dhisconnector/automation` in the backend. One for saving the automation toggle and the other one for submit automation data.  The frontend has 2 submit buttons in the same form for those two options. So we need to add the params to differentiate the 2 buttons/options.